projects/1.16/assets/minecolonies/minecolonies/lang/zh_cn.json

```
@@ -608,13 +618,15 @@
   "com.minecolonies.coremod.permission.explode": "爆炸",
   "com.minecolonies.coremod.permission.hurt_citizen": "伤害市民",
   "com.minecolonies.coremod.permission.hurt_visitor": "伤害游客",
+  "com.minecolonies.coremod.permission.map_border": "在地图上显示边界",
+  "com.minecolonies.coremod.permission.map_deaths": "在地图上显示死亡信息",
   "com.minecolonies.coremod.permission.receive_messages_far_away": "远方的信息",
   "com.minecolonies.coremod.permission.can_keep_colony_active_while_away": "保持殖民地加载",
-  "item.supplycampdeployer.invalid": "需要一片至少 16×17 大的平坦地面，并且没有花和草",
+  "item.supplycampdeployer.invalid": "需要一片至少16×17大的平坦地面，并且没有花和草",
   "item.supplycampdeployer.invalid.water_block_needed": "只能水面上放置补给船，这些地方必须有水：%s。",
   "item.supplycampdeployer.invalid.lava_block_needed": "在下界放置补给船需要在船下方有熔岩。这些方块必须是熔岩：%s。",
-  "item.supplycampdeployer.invalid.solid_block_needed": "放置补给营地需要至少 16×17 的平坦地面，并且没有花和草。这些方块必须是实心的：%s",
-  "item.supplycampdeployer.invalid.air_block_needed": "放置补给营地需要至少 16×17 的平坦地面，并且没有花和草。这些方块必须是空的：%s",
+  "item.supplycampdeployer.invalid.solid_block_needed": "放置补给营地需要至少16×17的平坦地面，并且没有花和草。这些方块必须是实心的：%s",
+  "item.supplycampdeployer.invalid.air_block_needed": "放置补给营地需要至少16×17的平坦地面，并且没有花和草。这些方块必须是空的：%s",

```

```suggestion
  "item.supplycampdeployer.invalid.solid_block_needed": "放置补给营地需要至少16×17的平坦地面，并且没有花和草。这些方块必须是固体：%s",
  "item.supplycampdeployer.invalid.air_block_needed": "放置补给营地需要至少16×17的平坦地面，并且没有花和草。这些方块必须是空气：%s",
```
https://minecraft.fandom.com/zh/wiki/%E6%9D%90%E6%96%99/Java%E7%89%88#%E5%9B%BA%E4%BD%93%E6%96%B9%E5%9D%97